File: psup assignment/cpp/q19.cpp
#include <iostream>
#include <string>
#include <cctype>
int main() {
    int choice;
    std::cout << "\n--- Program 19: Fuel Type Check ---\n";
    std::cout << "Enter fuel type (1: Petrol, 2: Diesel, 3: Electric): ";
    std::cin >> choice;
    switch (choice) {
        case 1: std::cout << "Petrol" << std::endl; break;
        case 2: std::cout << "Diesel" << std::endl; break;
        case 3: std::cout << "Electric" << std::endl; break;
        default: std::cout << "Invalid choice" << std::endl; break;
    }
    return 0;
}

File: psup assignment/java/q19.java
import java.util.*;
class FuelType {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter fuel type (1: Petrol, 2: Diesel, 3: Electric): ");
        int choice = input.nextInt();
        switch (choice) {
            case 1: System.out.println("Petrol"); break;
            case 2: System.out.println("Diesel"); break;
            case 3: System.out.println("Electric"); break;
            default: System.out.println("Invalid choice"); break;
        }
        input.close();
    }
}